recipe/meta.yaml

```diff
@@ -19,24 +19,21 @@ build:
     - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
   entry_points:
     - amaranth-rpc = amaranth.rpc:main
-    - nmigen-rpc = nmigen.rpc:main
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
-    - setuptools
-    - setuptools_scm
+    - pdb-backend
```

Comment:
`pdm-backend`?

Reply:
I write a lot of bugs so the the fingers just wanted to type `pdb` 😢